examples/helpers/ldtk.rs

```diff
@@ -25,7 +25,7 @@ impl Plugin for LdtkPlugin {
 }
 
 #[derive(TypeUuid)]
-#[uuid = "e51081d0-6168-4881-a1c6-4249b2000d7f"]
+#[uuid = "abf9eaf2-f21c-4b46-89b0-8aa5c42199af"]
```

Comment:
Was this sharing a UUID from somewhere else? 🤔

Reply:
Just with the Tiled example's `TiledMap`. Not the most likely collision, but I wouldn't put it totally outside the realm of possibility.